src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -327,20 +365,29 @@ public async Task<SyncTriggersPayload> GetSyncTriggersPayload()
 
             // Add all listable functions details to the payload
             JObject functions = new JObject();
-            string routePrefix = await WebFunctionsManager.GetRoutePrefix(hostOptions.RootScriptPath);
+            // TODO move before L387 and remove Console.WriteLine
```

Comment:
Clean up todos and get code into final shape

Reply:
Sure. Will do it!

Reply:
Done